js/index.js

```diff
@@ -0,0 +1,110 @@
+import data from './appData.js';
+
+const menuBtn = document.getElementById('menu-btn');
```

Comment:
- [ optional] great job with the mobile menu. please ensure that when it pops up, the user should use only it to navigate or close it before continuing/been able to scroll through your page/site

![Screenshot (467)](https://user-images.githubusercontent.com/65298048/192098349-2acbe15e-6eb4-46c6-b473-75ffc6d647d8.png)